src/components/data-processing/MitochondrialContent/CalculationConfig.jsx

```diff
@@ -72,8 +72,6 @@ const CalculationConfig = (props) => {
 
   const filtering = false;
 
-  const activeMethod = config.method;
```

Comment:
I think you shouldn't remove this, just replace it's value with `absolute_threshold`